README.md

```diff
@@ -1 +1,52 @@
-# Hello-Microverse
\ No newline at end of file
+# Project Name
+
+> HELLO MICROVERSE.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Getting Started
+
+
```

Comment:
- [ ] Please add description to `Getting Started ` section by adding the steps of how to get a local copy up and running.